pyproject.toml

```diff
@@ -51,8 +51,14 @@ test = [
 # All packages
 all = ["ambersim[dev, test]"]
 
+[project.scripts]
+install-mujoco-from-src = "ambersim._scripts._install_shim:entrypoint"
+
 [tool.setuptools.packages.find]
-include = ["ambersim"]
+include = ["ambersim*"]
```

Comment:
@vincekurtz this was the bug, missing an asterisk here means I only imported the top-level of the package and not any of the subdirectories.